coalib/parsing/Globbing.py

```diff
@@ -29,6 +29,24 @@ def _end_of_set_index(string, start_index):
     return closing_index
 
 
+def glob_escape(string):
```

Comment:
`string` is a frequently used inbuilt module - https://docs.python.org/3/library/string.html

Would it be possible to rename this to something else so that it doesnt get confusing if we ever import that ?